README.rst

```diff
@@ -101,6 +101,15 @@ We’d love to get contributions from you! Once you are up and running,
 take a look at the `Contribution Document`_ to see how to get your
 changes merged in.
 
+Log Insight and vROps make outbound HTTP connections to your shim, with a JSON body containing alerts details.
+The shim is responsible for parsing the body and identifying key information relevant to its operation.
+The shim makes one or more upstream connections to some other service using a native library or raw HTTP, performing a needed task.
+
+At completion, the shim reports back an overall success or failure using the HTTP status code; :code:`200` is treated as success while all other results are treated as a failure.
```

Comment:
The shim is definitely a translator of the request, and is in a position to translate the response as well. Shims are always specific to the third-party, and are mapping the expectations of two products. For example, the shim could translate the string `success` or the lack of a `UserNotFoundException` into HTTP/200.

That said, the webhook originator accepting all 2xx as Success would definitely be reasonable too. If that's true, this text should change accordingly.